### ISSUE TITLE
Bug or misconfig?

### ISSUE THREAD

**Issue by smargold476** [spantaleev/matrix-docker-ansible-deploy#2185]
Hi all,

I use the integrated NGINX and Certbot so the renew of certificates should work without any extra configuration.

Initial creation of certificate works perfectly, but when the renew should happen I see the following errors in the ssl log:

`Certbot wasn't able to bind to :8080 using IPv4, this is often expected due to the dual stack nature of IPv6 socket implementations.`

and then

```
Certbot failed to authenticate some domains (authenticator: standalone). The Certificate Authority reported these problems:
  Domain: customdomain.my.domain
  Type:   unauthorized
  Detail: myip: Invalid response from https://customdomain.my.domain/login: "<!DOCTYPE html>\n<html class=\"ng-csp\" data-placeholder-focus=\"false\" lang=\"de\" data-locale=\"de_DE\" >\n\t<head\n data-requesttoken=\"q"

Hint: The Certificate Authority failed to download the challenge files from the temporary standalone webserver started by Certbot on port 8080. Ensure that the listed domains point to this machine and that it can accept inbound connections from the internet.
```

As the initial certificate creation works and `netstat -nap` tells me:
```
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN      4132/docker-proxy
tcp6       0      0 :::80                   :::*                    LISTEN      4138/docker-proxy
```
It looks like a problem directly related to renewal?

Maybe you can help me or point to a config issue?

Regards

**Comment by smargold476** [comment]
damn, looks like i solved my own issue:
should have checked the [docs](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-ssl-certificates.md)...:

> For automated certificate renewal to work, each port 80 vhost for each domain you are obtaining certificates for needs to forward requests for /.well-known/acme-challenge to the certbot container we use for renewal.
> 
> See how this is configured for the matrix. subdomain in /matrix/nginx-proxy/conf.d/matrix-synapse.conf Don't be alarmed if the above configuration file says port 8080, instead of port 80. It's due to port mapping due to our use of containers.

I'll double-check if that solves my issue.

**Comment by smargold476** [comment]
solved, updated the docs #2187
